src/state_manager/mod.rs

```diff
@@ -1313,6 +1314,35 @@ where
         )
     }
 
+    pub fn verified_client_status(
+        self: &Arc<Self>,
+        addr: &Address,
+        ts: &Arc<Tipset>,
+    ) -> anyhow::Result<Option<DataCap>> {
+        let id = self.lookup_id(addr, ts)?.expect("actor not found");
+        let network_version = self.get_network_version(ts.epoch());
+
+        // This is a copy of Lotus code, we need to treat all the actors below version 9
+        // differently. Which maps to network below version 17.
+        if (u32::from(network_version.0)) < 17 {
+            let act = self
+                .get_actor(&Address::VERIFIED_REGISTRY_ACTOR, *ts.parent_state())
+                .map_err(|e| Error::State(e.to_string()))?
```

Comment:
I don't understand what `wibni` means. 

It's a nice refactoring, I'll add this as an issue for another PR, because we have quite a bit of occurrences using `Error::State(blah.to_string())` pattern and I'd rather not have this change in this PR.

Reply:
would be nice if :)